src/code/system_malloc.c

```diff
@@ -6,6 +6,16 @@
 
 Arena gSystemArena;
 
+#if PLATFORM_N64
+#define DECLARE_INTERRUPT_MASK OSIntMask __mask;
```

Comment:
imo the semicolon should be outside of the macro

Reply:
I tried, couldn't get it to work for GC :/